nengo_viz/components/raster.py

```diff
@@ -10,6 +10,9 @@
 class Raster(Component):
     def __init__(self, viz, config, uid, obj, n_neurons=None):
         super(Raster, self).__init__(viz, config, uid)
+        if getattr(obj, 'neuron_type') is not None:
```

Comment:
If it won't be none, then I think checking the probeable list generalizes better than checking for a specific type (Direct()) . 

Any thoughts?